components/QuizForm.tsx

```diff
@@ -30,10 +42,29 @@ const QuizForm: React.FC<Props> = ({ isLoading, questionSet, handleNextQuestion
     setShowCorrectAnswer(true);
   };
 
+  const handleNext = () => {
+    reset();
+    setShowCorrectAnswer(false);
+    setQuestionsSolved(questionsSolved + 1);
+    handleNextQuestion();
+  };
+
   return (
-    <form onSubmit={handleSubmit(onSubmit)}>
+    <div className="relative">
```

Comment:
Don't remove the form tag
```suggestion
    <form onSubmit={handleSubmit(onSubmit)}>
```